src/ontology/cl-edit.owl

```diff
@@ -30948,7 +30952,7 @@ SubClassOf(obo:CL_4030039 ObjectSomeValuesFrom(obo:RO_0000053 obo:PATO_0070006))
 
 # Class: obo:CL_4030040 (endometrial ciliated epithelial cell)
 
-AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:31290979") Annotation(oboInOwl:hasDbXref "PMID:37999981") Annotation(oboInOwl:hasDbXref "doi:10.1038/s41591-020-1040-z") Annotation(oboInOwl:hasDbXref "https://www.proteinatlas.org/humanproteome/single+cell+type/Glandular+epithelial+cells#endometrialciliatedcells") Annotation(oboInOwl:hasDbXref "doi:10.1016/j.coemr.2024.100503") obo:IAO_0000115 obo:CL_4030040 "A multi-ciliated cell of the endometrial epithelium. This cell is characterized by the presence of 9+2 motile cilia on its apical surface, which facilitates the movement of mucus across the endometrial surface.")
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:31290979") Annotation(oboInOwl:hasDbXref "PMID:37999981") Annotation(oboInOwl:hasDbXref "doi:10.1016/j.coemr.2024.100503") Annotation(oboInOwl:hasDbXref "doi:10.1038/s41591-020-1040-z") Annotation(oboInOwl:hasDbXref "https://www.proteinatlas.org/humanproteome/single+cell+type/Glandular+epithelial+cells#endometrialciliatedcells") obo:IAO_0000115 obo:CL_4030040 "A multi-ciliated cell of the endometrial epithelium. This cell is characterized by the presence of 9+2 motile cilia on its apical surface, which facilitates the movement of mucus across the endometrial surface.")
```

Comment:
you can change this reference DOI: [10.1016/j.coemr.2024.100503](https://doi.org/10.1016/j.coemr.2024.100503) for its respective PMID: 38293616 and this reference DOI: [10.1038/s41591-020-1040-z](https://doi.org/10.1038/s41591-020-1040-z) for its respective PMID: 32929266